pr_agent/git_providers/gitlab_provider.py

```diff
@@ -87,8 +87,12 @@ def get_files(self):
         return [change['new_path'] for change in self.mr.changes()['changes']]
 
     def publish_description(self, pr_title: str, pr_body: str):
-        logging.exception("Not implemented yet")
-        pass
+        try:
+            self.mr.title = pr_title
+            self.mr.description = pr_body
+            self.mr.save()
+        except Exception as e:
+            logging.exception(f"Could not update merge request {self.id_mr} description: {e}")
 
     def publish_comment(self, mr_comment: str, is_temporary: bool = False):
```

Comment:
Instead of catching a general Exception, it would be better to catch specific exceptions that you expect might be raised in the try block. This way, unexpected exceptions won't be silently ignored and can be properly dealt with. [important]